example.cpp

```diff
@@ -82,7 +84,7 @@ TEST_CASE_METHOD(CoherentUnitTestFixture, "test_incsc")
     }
 }
 
-TEST_CASE_METHOD(CoherentUnitTestFixture, "test_decsc")
+/*TEST_CASE_METHOD(CoherentUnitTestFixture, "test_decsc")
```

Comment:
Style: put the /* on a line by itself in advance of the section of code it’s commenting out.

Reply:
And likewise for the trailing `*/` on a line by itself afterwards.